march_joint_inertia_controller/src/inertia_estimator.cpp

```diff
@@ -152,23 +152,15 @@ void InertiaEstimator::inertia_estimate()
   correlation_calculation();
   K_i_ = gain_calculation();
   K_a_ = alpha_calculation();
-  torque_e = filtered_joint_torque_[0] - filtered_joint_torque_[1];
-  acc_e = filtered_acceleration_array_[0] - filtered_acceleration_array_[1];
+  const double torque_e = filtered_joint_torque_[0] - filtered_joint_torque_[1];
+  const double acc_e = filtered_acceleration_array_[0] - filtered_acceleration_array_[1];
   joint_inertia_ = (torque_e - (acc_e * joint_inertia_)) * K_i_ * K_a_ + joint_inertia_;
 }
 
 // Calculate the alpha coefficient for the inertia estimate
 double InertiaEstimator::alpha_calculation()
 {
-  double vib = vibration_calculation();
-  if (vib < min_alpha_)
-  {
-    vib = min_alpha_;
-  }
-  else if (vib > max_alpha_)
-  {
-    vib = max_alpha_;
-  }
+  double vib = std::max(std::min(vibration_calculation(), min_alpha_), max_alpha_);
```

Comment:
I think you want it the other way around with the min and max but what you really want is std::clamp.